### ISSUE TITLE
Keep the IOS app loge in

### ISSUE THREAD

**Issue by rporczyn** [spreedbox/spreedbox#12]
Do you have any idea how to keep user logged in on IOS?
Every time a would like to use app I need to logged in and I can receive calls only if I'm logged in.

It would work in the way that the app is still logged in bud access to this app is only protected by PIN or any other IOS feature.

Do you plan to change it ?

**Comment by Ivansss** [comment]
User should be logged in while the app is running (foreground or background).
If the app is closed completely, user will be logged out. But that's intentional.

**Comment by leonklingele** [comment]
@Ivansss Why is this done intentionally?

**Comment by Ivansss** [comment]
It was a decision made due to security reasons.
To build a system which include that feature, some user info should be stored.

**Comment by leonklingele** [comment]
@Ivansss does the Nextcloud Desktop / iOS app store user credentials?

**Comment by Ivansss** [comment]
@leonklingele Do you mean Spreed.ME iOS app?

**Comment by leonklingele** [comment]
@Ivansss no, does the Nextcloud Desktop / iOS store user credentials? If yes, what's the problem with storing them in the Spreed.ME iOS app?

**Comment by Ivansss** [comment]
@leonklingele I guess they do. Spreed.ME iOS app also stores user credentials in order to reconnect the last logged in user when the app is relaunched.
What I meant in my first comment is that if a user is not running the app (foreground or background), it won't receive any call or message.

**Comment by leonklingele** [comment]
> Spreed.ME iOS app also stores user credentials

Has this always been the case? As far as I can remember, it didn’t store user credentials when this issue was opened.

@rporczyn can you please verify that the login information is now kept across Spreed.ME app-restarts?
